fd/file.d.go

```diff
@@ -244,12 +244,15 @@ func (f *FileD) getStaticInfo(pipelineConfig *cfg.PipelineConfig, pluginKind pip
 	return &infoCopy, nil
 }
 
-func (f *FileD) Stop() {
+func (f *FileD) Stop(ctx context.Context) error {
 	logger.Infof("stopping pipelines=%d", len(f.Pipelines))
+	err := f.server.Shutdown(ctx)
 	_ = f.server.Shutdown(context.TODO())
```

Comment:
Why is server.shutdown called twice?  why context.TODO in second call?

Reply:
Merging master gone wrong. Fixed.